.gitignore

```diff
@@ -1,4 +1,7 @@
+.vscode
+.DS_Store
 bazel-bin
 bazel-out
 bazel-eventuals
-bazel-testlogs
\ No newline at end of file
+bazel-testlogs
+bazel-stout-eventuals
```

Comment:
That's an old directory that you can delete on your local machine and should not be generated again in the future!